pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_assigning-metadata-to-a-module.adoc

```diff
@@ -0,0 +1,30 @@
+[id="assigning-metadata-to-a-module_{context}"]
+= Assigning metadata to a module
+
+Assign metadata to a module for search engine optimization (SEO). SEO improves the findability of modules.
+
+.Prerequisites
+
+* You are logged in.
+* The module to be assigned metadata has been imported to {ProductShortName}.
+
+.Procedure
+
+. Navigate to link:{LinkToSearchPage}[Search page].
+
+. Click the name of the module you want to assign metadata.
++
+If you do not see the module listed, search for the modules using the search box.
+
+. In the list of module versions, click the three vertical dots next to the version of the module you want to assign metadata.
+
+. Click *Edit metadata* to assign the following metadata: 
++
+* *Product Name* - Select the Product and its Version that the module should be published for.
```

Comment:
"Product" -> "product"
"Version" -> "version"

Unless you are referring to the UI. But you should refer to a UI element or a command as an element, not as a word or term in a sentence.

```suggestion
* *Product Name* - Select the product and product version for the module.
```